tests/factories/factories.py

```diff
@@ -38,6 +38,16 @@ def TeamFactory():
     return response['data']['selected_team']
 
 
+def OrguserPkFactory(org_pk=None):
```

Comment:
this should return an orguser, OrguserFactory would be much more useful

Reply:
OrguserFactory already exist, it create a new org_user and return this user.
Should i create a new method in test_helper.py instead of new factory class ?